src/main/java/com/google/sps/api/tasks/TasksClientAdapter.java

```diff
@@ -54,6 +55,9 @@ public TasksClientAdapter() throws IOException {
    */
   public List<Task> getTasks(String tasksListId) throws IOException {
     List<Task> tasks = tasksClient.tasks().list(tasksListId).execute().getItems();
+    if (tasks == null) {
+      return new ArrayList<>();
```

Comment:
Nice check for null.
You could consider returning Collections.emptyList(). If you do not plan to add tasks to this list, it might be better practice, as it doesn't create new instance, and I find it a bit more expressive.